src/cmds/sys/uname.c

```diff
@@ -61,10 +60,14 @@ int main(int argc, char *argv[]) {
 		}
 	}
 
-	if (!uname_args_not_empty(&args))
+	if (!uname_args_not_empty(&args)) {
 		args.with_s = 1;
+	}
+
+	if (-1 == uname(&info)) {
+		printf("uname failed with code error %d", errno);
```

Comment:
Should function `return 1;` here to avoid access to `info`?

Reply:
thanks, done